CHANGES.md

```diff
@@ -1,3 +1,8 @@
+v2.1.3 (05-Aug-2024)
+====================
+- Now all subdirectories of the ``styles`` folder are parsed for styles
+- Added all 23 "discrete rainbow" styles proposed by Paul Tol on his website
```

Comment:
Can you merge these entries to the `v2.1.2` section? That will be the next release.

Reply:
I can certainly do so